bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -2829,6 +2830,7 @@ void BCStateTran::checkFirstAndLastCheckpoint(uint64_t firstStoredCheckpoint, ui
 
 void BCStateTran::checkReachableBlocks(uint64_t genesisBlockNum, uint64_t lastReachableBlockNum) {
   if (lastReachableBlockNum > 0) {
+    LOG_INFO(logger_, "");
```

Comment:
Instead of putting empty logs, can we log input parameters?

Reply:
Yes, I can do that, although almost all parameters are already printed outside in checkConsistency(), it would be more convenient.
Thanks.